docs/user_v2/how-to/make-a-device.rst

```diff
@@ -1,4 +1,84 @@
+.. note::
+
+    Ophyd.v2 is included on a provisional basis until the v2.0 release and 
+    may change API on minor release numbers before then
+
 Make a Device
 =============
 
-TODO
\ No newline at end of file
+.. currentmodule:: ophyd.v2.core
+
+To make a Device using Ophyd.v2, you need to subclass from the `Device`
+class, create some `Signal` instances, implement :meth:`~Device.connect()` and
+implement some other suitable Bluesky `hardware_interface`.
```

Comment:
How would I implement a non-EPICS device? By making special signal variants? I think there is enough interest in that to warrant a call-out in the docs somewhere.

Reply:
Good call, I think we should make a How-To guide about this. Can you think of a simple usecase for this? @untzag is `yaqc-bluesky` built on top of Ophyd v1? If so, do you think this might be a usecase to take inspiration from?

Reply:
My intuition says that this is a good source of inspiration, but also that a maximally simple example would suffice, e.g. a signal that reads and write to a file handle.

Reply:
I started making a `FileSignal` and `read()` and `set()` were easy, but `subscribe` less so. I can abstract some helpers from `EpicsSignal` but don't want to abstract too early, so I'd prefer to defer this until we've done Tango support...